File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

a = int(input('Give me a number: '))
star = '*'
empty = ' '
for i in range(1, a, 2):
    y = (a-i)//2
    print( y * empty + i * star)

File: week-07/day-01/draw-pyramid.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
// ---*---
// --***--
// -*****-
// *******
//
// The pyramid should have as many lines as lineCount is

let star = "*";
let space = " ";

for (let i = 0; i < lineCount; i++){ 
    let y = i + 1;
    console.log(space.repeat(lineCount - y) + star.repeat(i * 2 + 1) + space.repeat(lineCount - y));
}

